tools/serverpod_cli/bin/generator/class_analyzer.dart

```diff
@@ -658,4 +668,20 @@ class ClassAnalyzer {
 
     return true;
   }
+
+  bool _validateAndParseServerSideOnly(
+      YamlMap documentContents, CodeAnalysisCollector collector) {
+    var serverSideOnlyNode = documentContents.nodes['serverSideOnly'];
```

Comment:
The name is (obviously) open for debate.

Reply:
I think `serverOnly` seems a bit more concise.

Reply:
https://github.com/serverpod/serverpod/pull/625/commits/c7d86bb282299e11024d72023c2ee83abe6d66c0